### PR TITLE
Feat/mariner bash

### PR DESCRIPTION
**Pull request by mattgarvin1**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### New Features
- enable gen3-fuse to work with mariner

### CODE REVIEW

File: marinerRun.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+trap cleanup SIGTERM
+if [ $MARINER_COMPONENT == "engine" ]; then
+  echo "waiting for engine to finish.."
+  while [[ ! -f /$ENGINE_WORKSPACE/workflowRuns/$RUN_ID/done ]]; do
+    :
+  done
+else
+  echo "waiting for task to finish.."
+  while [[ ! -f $TOOL_WORKING_DIR\done ]]; do
```
Comment (ZakirG) [inline marinerRun.sh:36]: can you link to the code where this environment variable is defined?

Comment (ZakirG) [reply marinerRun.sh:36]: and also -- to the code where this script is used?

Comment (mattgarvin1) [reply marinerRun.sh:36]: Yes, here's the spot: https://github.com/uc-cdis/mariner/blob/master/mariner/k8s.go#L219-L224

Comment (mattgarvin1) [reply marinerRun.sh:36]: mariner config, has the command for the gen3-fuse container: https://github.com/uc-cdis/gitops-dev/blob/master/mattgarvin1.planx-pla.net/manifest.json#L256-L259

and that command gets pulled out of the config here: https://github.com/uc-cdis/mariner/blob/b20a2fba97d16d044ce276f01b9cba03e49f3b7c/mariner/k8s.go#L532-L544

as part of this function that constructs the gen3-fuse container spec within mariner: https://github.com/uc-cdis/mariner/blob/b20a2fba97d16d044ce276f01b9cba03e49f3b7c/mariner/k8s.go#L68-L74

**Pull request merged**